lexers/lexers_test.go

```diff
@@ -64,15 +70,21 @@ func TestLexers(t *testing.T) {
 			actual, err := chroma.Tokenise(lexer, nil, string(actualText))
 			assert.NoError(t, err)
 
-			// Read expected JSON into token slice.
-			var expected []chroma.Token
-			r, err := os.Open(expectedFilename)
-			assert.NoError(t, err)
-			err = json.NewDecoder(r).Decode(&expected)
-			assert.NoError(t, err)
+			if *write {
```

Comment:
Allows you to run `go test ./... -write` to write the current lexer output to the expected file. I left this in because I found it very useful for making changes, happy to take it out or put it in another PR, just let me know.

Reply:
Awesome!

Can you update lexers/testdata/README.md with these new instructions?

Reply:
Ha! I didn't even see the readme. That makes sense I'll delete this change.

Reply:
Nooo! I prefer this approach! It just needs to be documented.

Reply:
I'll push up a separate pr later this afternoon.